1-js/99-js-misc/04-reference-type/article.md

```diff
@@ -1,15 +1,14 @@
-
```

Comment:
You removed first line. please add it back.

if you're using a code editor, disable all formatters for `.md` files to prevent future issues.